app/src/main/java/com/rahobbs/todo/helpers/TodoItem.java

```diff
@@ -0,0 +1,88 @@
+package com.rahobbs.todo.helpers;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/1/16.
+ * Model for a single to-do item
+ */
+public class TodoItem {
```

Comment:
excellent job keeping this class super lean and not inflating it with unnecessary helpers / related static methods!